scripts/sidecar-target-triple.js

```diff
@@ -0,0 +1,35 @@
+import { execa } from 'execa'
+import { existsSync, renameSync } from 'fs'
+
+const ext = process.platform === 'win32' ? '.exe' : ''
+const bin = 'benchttp-server'
+const bindir = 'src-tauri/bin'
+const oldPath = `${bindir}/${bin}${ext}`
+
+function suffixBinary(targetTriple) {
+  return `${bindir}/${bin}-${targetTriple}${ext}`
+}
+
+async function getTargetTriple() {
+  const rustInfo = (await execa('rustc', ['-vV'])).stdout
+  return /host: (\S+)/g.exec(rustInfo)[1]
+}
+
+async function main() {
+  const targetTriple = await getTargetTriple()
+
+  if (!targetTriple) {
+    console.error('Error: Failed to determine platform target triple')
+    process.exit(1)
+  }
```

Comment:
Instead of handling errors partly here and partly in the catch block L33-34, why not just throw here and handle in the catch block?

```suggestion
  if (!targetTriple) {
    throw new Error('Failed to determine platform target triple')
  }
```